test/lit/passes/local-subtyping.wast

```diff
@@ -229,6 +234,40 @@
     )
   )
 
+  ;; CHECK:      (func $multiple-iterations-refinalize-call-ref (type $i32_=>_none) (param $i i32)
+  ;; CHECK-NEXT:  (local $f (ref $ret-i31))
+  ;; CHECK-NEXT:  (local $x i31ref)
+  ;; CHECK-NEXT:  (local.set $f
+  ;; CHECK-NEXT:   (ref.func $ret-i31)
+  ;; CHECK-NEXT:  )
+  ;; CHECK-NEXT:  (local.set $x
+  ;; CHECK-NEXT:   (call_ref $ret-i31
+  ;; CHECK-NEXT:    (local.get $f)
+  ;; CHECK-NEXT:   )
+  ;; CHECK-NEXT:  )
+  ;; CHECK-NEXT: )
+  (func $multiple-iterations-refinalize-call-ref (param $i i32)
+    (local $f (ref null $ret-any))
+    (local $x (anyref))
+    (local.set $f
+      (ref.func $ret-i31)
+    )
+    (local.set $x
+      ;; After $f is refined to hold $ret-i31 and the call_ref is refinalized,
+      ;; we will be able to refine $x to i31.
+      (call_ref $ret-any
+        (local.get $f)
+      )
+    )
+  )
```

Comment:
Please add a test where `$f` is refined to a bottom type.

Reply:
Good idea, done.